CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -595,53 +597,91 @@ private void createNote(final NotesViewEntry newNote) {
 
         final String noteColour = newNote.isUserCreated() ? USER_COLOUR : AUTO_COLOUR;
 
+        // Define dateTime label
         final Label dateTimeLabel = new Label((new SimpleDateFormat(DATETIME_PATTERN).format(new Date(Long.parseLong(newNote.getDateTime())))));
         dateTimeLabel.setWrapText(true);
-        dateTimeLabel.setStyle("-fx-font-weight: bold; " + fontStyle);
+        dateTimeLabel.setStyle("-fx-font-weight: bold;" + fontStyle);
 
+        // Define title text box
+        final TextField titleText = new TextField(newNote.getNoteTitle());
+        titleText.setStyle("-fx-font-weight: bold;"); // -fx-text-fill: black; -fx-background-color: white;");
+
+        // Define title label
         final Label titleLabel = new Label(newNote.getNoteTitle());
         titleLabel.setWrapText(true);
-        titleLabel.setStyle("-fx-font-weight: bold; " + fontStyle);
+        titleLabel.setStyle("-fx-font-weight: bold;" + fontStyle);
 
+        // Define content label
         final Label contentLabel = new Label(newNote.getNoteContent());
         contentLabel.setWrapText(true);
+        contentLabel.setMinWidth(50);
+        contentLabel.setAlignment(Pos.TOP_LEFT);
+
+        // Define content text area
+        final TextArea contentTextArea = new TextArea(newNote.getNoteContent());
+        contentTextArea.setWrapText(true);
 
         final VBox noteInformation;
 
+        // Define selection label
+        String selectionLabelText = "";
+        final Label selectionLabel = new Label(selectionLabelText);
+
         // If the note is user created add the selection details.
         if (newNote.isUserCreated()) {
-            String selectionLabelText;
             if (newNote.isGraphAttribute()) {
-                selectionLabelText = "Note selection is linked to the graph.";
+                selectionLabelText = "Note is linked to: the graph.";
```

Comment:
How about "Note linked to: Graph" to reduce the amount of words we have to see here on the note?

Reply:
Yep I agree, ill change it.